Samples/Interactivity/Commands/InteractiveCommands.cs

```diff
@@ -115,7 +118,7 @@ public async Task<IResult> SendDropdownAsync()
             {
```

Comment:
If only there was an ``MessageComponentBuilder`` based class that acts like ``IServiceCollection`` where it's used to add "components" that would eventually build into the same thing, however it would probably need an nested ``ActionRowBuilder``, ``SelectMenuBuilder``, etc (and builders for the other interaction components), Also have ``MessageComponentBuilder`` have an ``Build()`` method responsible for calling ``Build()`` for the other builders which will new up all of those components in order.

I think the builder option for messagecomponents would be an great addition to the Interactivity apis too as an alternative to newing them up ourselves (the builders could be made to throw on improper setup of each type of component to tell the developer of their bot that they are not doing something properly).